data/json/overmap/overmap_terrain/overmap_terrain_sunkenheli.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "overmap_terrain",
+    "id": [ "sunken_heli" ],
+    "name": "sunken heli",
+    "sym": "~",
+    "color": "light_blue",
+    "flags": [ "NO_ROTATE", "LAKE" ],
```

Comment:
Is there something I'm missing as to why this can't rotate?

Reply:
Oh right ye bullet trailer style sprites would be very cool and a PITA if it rotated ye